### DISCUSSION TITLE
Where I can find "1p3_million_mouse_brain.rds"  from sketch based tutorial?

### DISCUSSION THREAD

**Discussion by akauko** [satijalab/seurat#8344] [Q&A]
Hi,

I was doing tutorial on sketch based analysis at https://satijalab.org/seurat/articles/seurat5_sketch_analysis. However, I was not able to find required input file  "1p3_million_mouse_brain.rds". Where I could find this file?

**Comment by ruffleCSV** [comment]
did you find it ? if yes can you send the link please

**Comment by laoz-mskcc** [comment]
https://support.10xgenomics.com/single-cell-gene-expression/datasets/1.3.0/1M_neurons?

in Output files, 
Click "aggr - Gene/cell matrix HDF5(filtered)"

maybe need sign in first
